src/Fixer/CastNotation/ShortScalarCastFixer.php

```diff
@@ -33,9 +34,8 @@ public function getDefinition(): FixerDefinitionInterface
                     "<?php\n\$a = (boolean) \$b;\n\$a = (integer) \$b;\n\$a = (double) \$b;\n\$a = (real) \$b;\n\n\$a = (binary) \$b;\n",
                     new VersionSpecification(null, 7_03_99)
```

Comment:
i think CodeSample for <7.4 is not needed anymore too, and we can have similar detection for such samples

Reply:
It's not that straightforward, this is present in docs. We can try some approach... in separate PR 😉

Reply:
Will update docs and I wonder if we want to update the fixers too, take a look at https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7212